### ISSUE TITLE
[feature request] Add `watched_plate` config arg

### ISSUE THREAD

**Issue by robmarkcole** [robmarkcole/HASS-Sighthound#22]
Add `watched_plate` config arg which is a list of plates to watch for, with an entity created for each where the state is the time of the last detection of the plate. If a plate with the same number is matched, or similar by at least 4 characters, update this entity.

**Comment by alexandrechoske** [comment]
Hi @robmarkcole , how can I config this to detect cars? 
I didnt figure it out.

Thanks!

**Comment by robmarkcole** [comment]
@alexandrechoske please ask on the forums for this kind of general advice. Thanks

**Comment by jazzmonger** [comment]
Yes- that would fit the bill perfectly. If you make it generic, we can use it across other integrations that produce a 'result' that will be contained in a list of user provided results.
